.github/workflows/release.yml

```
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+      - name: Run tests
+        run: bundle exec spec 
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

```suggestion
        uses: softprops/action-gh-release@v2
```
Minor nit: Let's use the latest version of release action.